rados/write_step.go

```diff
@@ -23,11 +24,17 @@ type writeStep struct {
 }
 
 func newWriteStep(b []byte, writeLen, offset uint64) *writeStep {
+	bufPtr := unsafe.Pointer(&b[0])
 	return &writeStep{
 		b:         b,
-		cBuffer:   (*C.char)(unsafe.Pointer(&b[0])),
+		pg:        cutil.NewPtrGuard(bufPtr),
```

Comment:
I just noticed this comment. Does that mean PRs that would depend on this fix (e.g. https://github.com/ceph/go-ceph/pull/581) are blocked until after December's release?

Reply:
No, the PRs can be merged prior to release. We simply need to discuss our options, some of the work can possibly be split up, for example.  In fact I'd prefer to merge as much as possible sooner rather than later Unfortunately, things have been slow due to illnesses and time off. Please continue to ping us on these items so that we don't forget to work on them!